javascriptv3/example_code/README.rst

```diff
@@ -0,0 +1,43 @@
+###############################
+AWS SDK for JavaScript version 3 (v3) examples
+###############################
+
+These are examples for the AWS SDK for JavaScript (v3) public documentation.
+
+The prerelease version of the version 3 of the SDK is available `here <at https://github.com/aws/aws-sdk-js-v3>`_ .
+The 'SDK for JavaScript Developer Guide' for V3 is also scheduled for release later in 2020, and the topic containing this example will be hosted
+`here <https://docs.aws.amazon.com/sdk-for-javascript/v3/developer-guide/sqs-examples-managing-visibility-timeout.html`_ .
+
+**NOTE:** The AWS SDK for JavaScript (v3) is written in TypeScript so, for consistency, these examples are also in TypeScript. TypeScript extends of JavaScript so these examples can also be run as JavaScript.
+
+Prerequisites
+=============
+
+To build and run these examples, you'll need:
+
+- Set up the project environment to run these Node TypeScript examples, and install the required AWS SDK for JavaScript and third-party modules. Follow the instructions on in the README.md in the folder with the examples you're running. For for example, for an s3 example use the instructions `here <https://github.com/awsdocs/aws-doc-sdk-examples/blob/master/javascriptv3/example_code/s3/README.md>`_.
+- Create a shared configurations file with your user credentials. For more information about providing a shared credentials file, see `Loading credentials in Node.js from the shared credentials file <https://docs.aws.amazon.com/sdk-for-javascript/v3/developer-guide/loading-node-credentials-shared.html>`_ (release pending).
```

Comment:
release pending?

Reply:
Added 'Once it's released' phrasing throughout